Update/onik_005.txt

```diff
@@ -4327,15 +4343,15 @@ void main()
 //　つまりこれこそは擬態・カモフラージュの原点と言える＠
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "　つまりこれこそは擬態・カモフラージュの原点と言える。",
-		   NULL, "We'll call it Uncanny Valley Camouflage, if you will.", GetGlobalFlag(GLinemodeSp));
+		   NULL, "We'll call it \"Uncanny Valley Camouflage,\" if you will.", GetGlobalFlag(GLinemodeSp));
 	if (GetGlobalFlag(GADVMode)) { ClearMessage(); } else { OutputLineAll(NULL, "\n", Line_ContinueAfterTyping); }
 
 
 
 //　人間は相手を見て「それは人間だ」と認識して始めて行動に移れるのだ￥
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "　人間は相手を見て「それは人間だ」と認識して始めて行動に移れるのだ。",
-		   NULL, "When humans see others, they can only begin to act after confirming \"this is human.\"", Line_Normal);
+		   NULL, "When humans see others, they can only begin to act after confirming \"This is human.\"", Line_Normal);
```

Comment:
Yeah in this case I believe the quote is part of the sentence and so should not be separately capitalized.  It would be a valid sentence on its own but but isn't used that way here.  This also should apply to the rest of the cases as far as I know.